pkg/kubernetes/events/suite_test.go

```diff
@@ -0,0 +1,13 @@
+package events
+
+import (
+	"testing"
+
+	. "github.com/onsi/ginkgo"
```

Comment:
You don't need this anymore of you are using `testing` under this new folder

Reply:
This will be needed since for the time being we are continuing to allow Ginkgo based tests for external contributors, and haven't started enforcing the testing framework.